addons/crm/views/crm_lead_views.xml

```diff
@@ -501,13 +501,12 @@
                                     <div>
                                         <strong class="o_kanban_record_title"><field name="name"/></strong>
                                     </div>
+                                    <div class="o_kanban_record_subtitle">
```

Comment:
I would put the t-if on the div instead.
No reason not too.

Reply:
So no need to display Customer if no expected revenue?

Reply:
Oh you're right.
It is so badly formatted that I had not noticed.
Then maybe reformat to:
```xml
<div class="o_kanban_record_subtitle">
    <t t-if="record.planned_revenue.raw_value">
        <field name="planned_revenue" widget="monetary" options="{'currency_field': 'company_currency'}"/>
        <span t-if="record.partner_id.value"> - </span>
    </t>
    <span t-if="record.partner_id.value" t-esc="record.partner_id.value"></span>
</div>
```

For clarity?
Thanks for double checking!

Reply:
Perfect. Thanks to you!